Source/ProgressionSystemRuntime/Private/LevelActors/PSStarActor.cpp

```diff
@@ -70,7 +112,7 @@ void APSStarActor::TryPlayHideStarAnimation()
 	}
 
 	const FPSRowData& CurrentRow = UPSWorldSubsystem::Get().GetCurrentRow();
-	if (!CurrentRow.HideStarsAnimation)
+	if (!CurrentRow.MenuStarsAnimation)
```

Comment:
why you changed the HideStartsAnimation-check to be `MenuStarsAnimation`-check inside of the `HideStar` function? Seems mistake

Reply:
done as part of https://github.com/h4rdmol/ProgressionSystem/commit/a5231652e33f7f150dc6d860ded531e0a8493cb3